sql-plugin/src/main/303/scala/com/nvidia/spark/rapids/shims/spark303/RapidsShuffleInternalManager.scala

```diff
@@ -77,4 +77,4 @@ class ProxyRapidsShuffleInternalManager(conf: SparkConf, isDriver: Boolean)
     self.getReaderForRange(handle, startMapIndex, endMapIndex, startPartition, endPartition,
       context, metrics)
   }
-}
\ No newline at end of file
+}
```

Comment:
File shouldn't be modified, otherwise we need copyrights.  Yay, copyrights!  Applies to some other files as well that had trivial end-of-file whitespace changes.

Reply:
I don't know how to get rid of this. any ideas?

Reply:
wow this was trickier than I thought. Even after removing the eof, github wouldn't accept it.

I had to change the base branch and revert back to branch-22.04 base for it to work